src/util/manifest.js

```diff
@@ -14,6 +14,11 @@ const log = createLogger(__filename);
 export type ExtensionManifest = {
   name: string,
   version: string,
+  applications: {
```

Comment:
I changed this to `applications?` already upstream in https://github.com/mozilla/web-ext/pull/639